TACC/Core-Styles
# feat(elements): table, updates from tup-cms

## Overview

- Adds CMS styles for tables.
- Changes (minor) to basic table styles.

## Related

None.

## Changes

- remove border on first column
- change thead bottom border to be tbody top border
- add cms-specific styles
- (demo) add cms table variants

## Testing

1. Open http://localhost:3000/components/detail/table.
2. Open and review "… (CMS)" versus non-"… (CMS)" demos.
3. Verify "… (CMS)" styles:
    - add thick bottom border to table
    - add thick top border to headless tables
    - have larger text
    - have greater cell spacing
    - have greater text spacing (i.e. line-height)
4. Verify "Nested" table:
    - looks as it did before (compare to #125 "UI")

## UI

| | base | cms |
| - | - | - |
| basic | ![basic](https://user-images.githubusercontent.com/62723358/218875651-cc49d9b3-f9bf-4e4b-bf66-50a681e7a357.png) | ![basic (cms)](https://user-images.githubusercontent.com/62723358/218875648-a6b4a98e-b31d-466c-85fb-238fe88f102a.png) |
| via paragraphs | ![via paragraphs](https://user-images.githubusercontent.com/62723358/218875643-315efeba-dc2c-43ac-8354-ecccca4bf94f.png) | ![via paragraphs (cms)](https://user-images.githubusercontent.com/62723358/218875639-f0a31744-2f9a-4bec-b7ab-f832874d7a49.png) |
| no header | ![no header](https://user-images.githubusercontent.com/62723358/218875646-e2729855-daee-4975-8616-e0e3a700ca7e.png) | ![no header (cms)](https://user-images.githubusercontent.com/62723358/218875638-b1011e67-5101-440f-bc67-716aee44d42e.png) |

| | with paragraphs | nested |
| - | - | - |
| unchanged | ![with paragraphs](https://user-images.githubusercontent.com/62723358/218875644-0561fc54-1b89-4063-97cf-49cd49ae2b96.png) | ![nested](https://user-images.githubusercontent.com/62723358/218875632-559834ce-86f9-4637-989c-8dc0ec2b2eb8.png) |
